regtests/mww3_test_03/input/ww3_shel.inp

```diff
@@ -0,0 +1,29 @@
+$ WAVEWATCH III shell input file
```

Comment:
mww3_test03 is for multi setup even if it uses one single grid in it. Could you change it ww3_multi.inp and test it.